code/datums/wires/robot.dm

```diff
@@ -35,7 +35,7 @@
 					R.notify_ai(TRUE)
 		if(WIRE_CAMERA) // Pulse to disable the camera.
 			if(!isnull(R.camera) && !R.scrambledcodes)
-				R.camera.deactivate(usr, 0)
+				R.camera.toggle_cam(usr, 0)

```

Comment:
I renamed camera/proc/deactivate() to toggle_cam() since it's actually a toggle.